brewerdigital-llc/VivoCare
# Fix/vitals cards back buttons and onboarding screens

# Description
This PR:
- Uniforms back buttons
- Fixes back button flex bug that was causing some back buttons to extend across the width of the screen even though the icon only appears on one side of the screen. 
- Fixes bug causing vital cards to layout incorrectly after filter was added
- Creates OnboardingScreen component to DRY up those screens
- Fixes circular dependency warning
- Fixes content width warning for renderHTML
- Refactors instances of TouchableWithoutFeedback
  - I learned this morning that you don't need TouchableWithoutFeedback on ScrollViews, they have it built in by default. You only need to wrap areas of the screen that should close the keyboard when tapped that aren't within a ScrollView. I think that's where all the issues came from. Also, I'm not sure this was always the case...but maybe?

# Closes issue(s)
- Various fixes

# Steps to Test
1. Run app on real device
2. Test onboarding flow
3. Test "tap outside of keyboard" functionality throughout app to ensure expected behanvior.
4. See vital cards
5. Test back buttons - press to the far right of back buttons to ensure all flex bugs are fixed

# Changes include

# Checklist
